pkgs/development/python-modules/nettigo-air-monitor/default.nix

```diff
@@ -23,7 +23,7 @@ buildPythonPackage rec {
     owner = "bieniu";
     repo = pname;
     rev = "refs/tags/${version}";
-    hash = "sha256-K8EiDb6B18No9RNbw2a7U+FJQaXrrcFf0hgt40r6Igo=";
+    hash = "sha256-24O9Yl0+boxDtyPW4tBTsk2iDGGXf8ofkDHu8B+GxhE=";
   };
 
   propagatedBuildInputs = [
```

Comment:
I guess we don't need `orjson` anymore.
https://github.com/bieniu/nettigo-air-monitor/blob/727b9f2f23b81de266534db360c94b49afde4134/setup.py#L25